.github/workflows/test-playwright-and-deploy-to-fly-io.yml

```diff
@@ -26,19 +25,20 @@ jobs:
       - uses: pnpm/action-setup@v2
         with:
           version: 'latest'
+
       # Use the official setup-node action (sets up Node.js):
       # https://github.com/actions/setup-node
       - name: Use Node.js
         uses: actions/setup-node@v3
         with:
           node-version: 'lts/*'
           cache: 'pnpm'
+
       - name: Install dependencies
         run: pnpm install
       - name: Install SafeQL if not yet installed (eg. on Windows dev machines)
         run: grep package.json -e '@ts-safeql/eslint-plugin' || pnpm add @ts-safeql/eslint-plugin libpg-query
       - run: pnpm migrate up
-      # Also generates next-env.d.ts, required for tsc
```

Comment:
why was this removed?

Reply:
Reverted the deleted comment
[ebf3118](https://github.com/upleveled/next-js-example-spring-2023-vienna-austria/pull/15/commits/ebf3118340c41006824dcad6dd0d576059a1e4a0)